tests/pkg_littlefs/Makefile

```diff
@@ -3,4 +3,6 @@ include ../Makefile.tests_common
 USEMODULE += littlefs
 USEMODULE += embunit
 
+TEST_ON_CI_BLACKLIST += esp32-wroom-32
```

Comment:
Please add a comment as to why they are blacklisted (and that this should be undone ASAP)

Reply:
done: "# disabling due to unknown failure. See #15123."